templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List<a href="{% url 'taxi:manufacturer-create' %}"> +
```

Comment:
The `<h1>` tag is not properly closed. Ensure that the closing tag is correctly formatted to avoid rendering issues.